src/modules/tooltip.coffee

```diff
@@ -37,6 +37,8 @@ class Tooltip
     @container.style.left = Tooltip.HIDE_MARGIN
```

Comment:
I feel like messing with that functionality -- which is used for other logic -- is maybe out of scope for this PR, but would be nice to have for the future, for sure.